skfda/ml/classification/_depth_classifiers.py

```diff
@@ -104,3 +108,120 @@ def predict(self, X):
         ]
 
         return self.classes_[np.argmax(depths, axis=0)]
+
+
+class DDTransform(BaseEstimator, TransformerMixin):
+    r"""Depth-versus-depth (DD) transformer for functional data.
+
+    This transformer takes a list of k depths and performs the following map:
+
+    .. math::
+        \mathcal{X} &\rightarrow \mathbb{R}^G \\
+        x &\rightarrow \textbf{d} = (D_1^1(x),...,D_g^k(x))
+
+    Where :math:`D_i^j(x)` is the depth of the point :math:`x` with respect to
+    the data in the :math:`i`-th group using the :math:`j`-th depth of the
+    provided list.
+
+    Note that :math:`\mathcal{X}` is possibly multivariate, that is,
+    :math:`\mathcal{X} = \mathcal{X}_1 \times ... \times \mathcal{X}_p`.
+
+    Parameters:
+        depth_methods (default
```

Comment:
Maybe we could also allow passing one depth, as it seems a common case. In that case, maybe I would still call it `depth_method`.